### PR TITLE
Trash and archive

### PR DESCRIPTION
**Pull request by vermasavita**
###  What do you need to check?

- Delete Notes
- Archive Notes
- Delete from Archive
- Restore from Archive
- Delete Trash
- Restore from Trash
- 404 Error page
- Change favicon
- Refactor Code

Note: Ignore Filter commented code. I'll add that later

### CODE REVIEW

File: src/pages/trash/Trash.jsx [hunk 1]
```diff
@@ -9,12 +26,20 @@ const Trash = () => {
           <SideBar />
         </div>
         <div className="sub-container">
-          <div className="notes-category">
-            <h2>Trash Notes</h2>
-            <div className="saved-notes">
-              <TrashCard />
-            </div>
-          </div>
+          {trash.length !== 0 ? (
```
Comment (sanjitsarkar) [inline src/pages/trash/Trash.jsx:29]: Suggestion: You can add trash.length>0 and it will be more meaningful.

**Review by sanjitsarkar — APPROVED** [review]
Well written and readable code
Clean and Simple UI
Functionality is working as expected
Good to merge

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Code looks fine, was not able to test functionality because preview is not working for me, maybe you can cross check that.
Not able to create new note in preview

**Comment by vermasavita** [comment]
> Code looks fine, was not able to test functionality because preview is not working for me, maybe you can cross check that. Not able to create new note in preview

@Naman-Saxena1 Fixed that. Please check now and approve it.

**Review by Naman-Saxena1 — APPROVED** [review]

**Pull request merged**